scripts/predicts.py

```diff
@@ -0,0 +1,111 @@
+#retriever
+
+"""Retriever script for direct download of data data"""
+from retriever.lib.models import Table, Cleanup, correct_invalid_value
+from retriever.lib.templates import Script
+from retriever import VERSION
+from pkg_resources import parse_version
+
+
+class main(Script):
+    def __init__(self, **kwargs):
+        Script.__init__(self, **kwargs)
+        self.title = "PREDICTS Database"
+        self.name = "predicts"
+        self.ref = "http://data.nhm.ac.uk/dataset/902f084d-ce3f-429f-a6a5-23162c73fdf7"
+        self.urls = {"PREDICTS": "http://data.nhm.ac.uk/dataset/the-2016-release-of-the-predicts-database/resource/78dac1a9-6aa0-4dcb-9750-50df04f8ca6e/download"}
+        self.citation = "Lawrence N Hudson; Tim Newbold; Sara Contu; Samantha L L Hill et al. (2016). Dataset: The 2016 release of the PREDICTS database. http://dx.doi.org/10.5519/0066354"
+        self.keywords = ['biodiversity', 'anthropogenic pressures']
+        self.retriever_minimum_version = "2.0.dev"
+        self.version = "1.0.0"
+        self.description = "A dataset of 3,250,404 measurements, collated from 26,114 sampling locations in 94 countries and representing 47,044 species."
+
+        if parse_version(VERSION) <= parse_version("2.0.0"):
+            self.shortname = self.name
+            self.name = self.title
+
+    def download(self, engine=None, debug=False):
+        Script.download(self, engine, debug)
+        engine = self.engine
+
+        filename = "predicts_data.csv"
```

Comment:
It looks to me like the file the in archive is actually `database.csv` not `predicts_data.csv`. I get an error using the later, but things execute using the former.